README.md

```
@@ -0,0 +1,42 @@
+<p align="center">
+    <h1 align="center">Health AI</h1>
+    <p align="center">
+        <img align="center" width="70" src="./assets/icons/splash-icon.png"/>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: img

(MD033, no-inline-html)